<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_invite</class>
 <widget class="QDialog" name="Dialog_invite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="EnvoyerSMS">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer SMS</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>61</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Num Tel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Presence</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>77</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Mailing</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Recherche">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Modifier">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>83</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ajouter">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>83</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Supprimer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>118</width>
     <height>152</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="Linecin"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineNom"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LinePrenom"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineNumTel"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Linepresence"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEmail"/>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>751</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TriNom">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri Nom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TriID">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineRecherche">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>116</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
